<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserFile_ui</class>
 <widget class="QDialog" name="EditUserFile_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#label_picture
{
	
	border-image: url(:/new/prefix1/C:/Users/z/Desktop/icon_image/user.png);
}

#frame
{
	
	background-color: rgb(255, 255, 255);
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="3">
       <widget class="QLabel" name="label_picture">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/prefix1/m_icon/user.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>真实姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lETrealName">
          <property name="placeholderText">
           <string>请输入真实姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_score">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>证件号码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lETIDnum">
          <property name="placeholderText">
           <string>请输入有效证件号码（居民身份证、护照等）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_email">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>用户邮箱：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETEmail">
          <property name="placeholderText">
           <string>请输入您的邮箱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelpic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>头像上传：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETPic">
          <property name="placeholderText">
           <string>请上传png格式文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 40);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_language">
          <property name="text">
           <string>语言资质：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="placeholderText">
           <string>示例：2019年11月TOEFL119分、大学英语专业8级等</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_Proved_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>语言资质证明上传：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETPicLanguage">
          <property name="placeholderText">
           <string>请上传png格式文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 40);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 50);
font: 11pt &quot;AcadEref&quot;;</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
